roles/provision-metrics-apb/defaults/main.yml

```diff
@@ -1,30 +1,30 @@
 # Prometheus Values
-prometheus_image: "registry.access.redhat.com/openshift3/prometheus"
+prometheus_image: "docker.io/openshift/origin-prometheus"
 prometheus_version: "v3.11"
 prometheus_port: 9090
 prometheus_proxy_port: 4180
 
 # Grafana values
-grafana_image: "registry.access.redhat.com/openshift3/grafana"
-grafana_version: "v3.11"
+grafana_image: "docker.io/aerogear/grafana"
+grafana_version: "5.4.3"
 grafana_port: 3000
 grafana_proxy_port: 4181
 grafana_log_level: debug
 grafana_admin_username: admin
 
 # Api server values
 api_server_image: "docker.io/aerogear/aerogear-app-metrics"
-api_server_version: "0.0.12"
+api_server_version: "0.0.13"
```

Comment:
For some reason upgrading to this release breaks the tests. :-1:

Reply:
@psturc the error didn't make sense to me. can you have a look?

Reply:
It fails on pulling grafana image.
```
Failed to pull image "docker.io/aerogear/grafana:5.4.3": rpc error: code = 2 desc = Tag 5.4.3 not found in repository docker.io/aerogear/grafana
```
We have only v5.0.0 available in [aerogear dockerhub org](https://hub.docker.com/r/aerogear/grafana/tags)

Reply:
Hmmmmm, I am quite confused. :thinking: 

@psturc what you're saying makes sense, _but_ the erroneous image tag for grafana was added in the previous commit, for which the tests _passed_. See the "Commits" tab:

![image](https://user-images.githubusercontent.com/442386/52555160-d6bac780-2de0-11e9-8612-10402817a4c4.png)

Shouldn't that also have failed?

Reply:
@grdryn yeah, it should have also failed. But for some reason, if running `apb test` takes more than some time (I don't know the exact time limit), it does not return the error message we're checking [here](https://github.com/aerogearcatalog/metrics-apb/blob/master/Jenkinsfile#L49). So we end up with incorrectly evaluated test result. I fixed it already in [sync-app-apb](https://github.com/aerogearcatalog/sync-app-apb/blob/master/Jenkinsfile#L54). I will do the same in this repo as well.

Reply:
@psturc ah ok, thanks! I've just updated now to see if `docker.io/grafana/grafana` will work better (it's what OpenShift use upstream). :+1: